### PR TITLE
[Console][Testing] Correct what appears to be intended behavior vs actual behavior in do…

### PR DESCRIPTION
**Pull request by ghost**
…cumentation

When testing a command with an `InputOption::VALUE_NONE` option, and symfony/console v5.4.15 I have to provide a non-empty value to set the flag to the 'on' value. Perhaps this is changed in a later version?

### CODE REVIEW

File: console.rst [hunk 1]
```diff
@@ -537,11 +537,11 @@ call ``setAutoExit(false)`` on it to get the command result in ``CommandTester``
 
 .. caution::
 
-    When testing ``InputOption::VALUE_NONE`` command options, you must pass an
+    When testing ``InputOption::VALUE_NONE`` command options, you must pass a non
     empty value to them::
```
Comment (wouterj) [inline console.rst]:
```suggestion
    When testing ``InputOption::VALUE_NONE`` command options, you must pass ``true``
    to them::
```

File: console.rst [hunk 2]
```diff
@@ -537,11 +537,11 @@ call ``setAutoExit(false)`` on it to get the command result in ``CommandTester``
...
     
         $commandTester = new CommandTester($command);
-        $commandTester->execute(['--some-option' => '']);
+        $commandTester->execute(['--some-option' => 'true']);
```
Comment (wouterj) [inline console.rst]:
```suggestion
        $commandTester->execute(['--some-option' => true]);
```

**Review by wouterj — APPROVED** [review]
Looking at some of Symfony's own tests, I think you're correct. However, Symfony's own tests always use `true` (the boolean), so let's stick with that in the documentation as well.

**Comment by javiereguiluz** [comment]
Good catch Tom! Thanks and congrats on your first (and nice) Symfony Docs contribution 🎉 

Note: I'm closing this PR in favor of #19580, which contains the same commits but fixes the merge conflicts. Thanks.

**Pull request closed without merging**